pipelines/performance-tests-pipeline.yml

```diff
@@ -308,6 +308,35 @@ jobs:
 
               echo "\"@three-hundred-and-fifty-thousand\"" | gsutil cp - gs://census-rm-performance-sample-files/target-scenario-tag.txt || true
 
+- name: "Select Test - Receipting"
+  plan:
+  - *slack_performance_tests_started
```

Comment:
This is a bit of a bug that I introduced. This should be a totally different slack alert, but I messed up. If you like, you can fix it!

Reply:
Got rid of these slack alerts